### DISCUSSION TITLE
innerHTML => createElement로 수정

### DISCUSSION THREAD

**Discussion by mixnuts211** [Already-12-O-clock/ABCDmoviE#96] [Ideas]
### innerHTML는 아래와 같은 단점을 가지고 있습니다.
- 요소 노드의 innerHTML 프로퍼티에 할당한 HTML 마크업 문자열은 렌더링 엔진에 의해 파싱되어 요소 노드의 자식으로 DOM에 반영되고, 이때 사용자로부터 입력받은 데이터를 그대로 innerHTML 프로퍼티에 할당하는 것은 크로스 사이트 스크립팅 공격(XSS)에 취약하므로 위험합니다.
- 요소 노드의 모든 자식 노드를 제거하고 할당한 HTML 마크업 문자열을 파싱하여 DOM을 변경합니다.
- 새로운 요소를 삽입할 때 삽입될 위치를 지정할 수 없습니다.
> 출처 - 모던 자바스크립트 Deep Dive
> 참고 - https://developer.mozilla.org/ko/docs/Web/API/Element/innerHTML

이에 따라 요소 삽입을 위해 사용된 `innerHTML`를 `createElement`로 수정하려고 합니다.

**Comment by July249** [comment]
`createElement`로 요소를 생성하여 직접 `innerHTML`과 동일한 DOM 트리 생성을 진행하는 것이 이런 부분에서 차이가 있군요!

말씀해주신 3번째 단점을 해결하는 과정에서 직접 요소를 생성하는 것이 확장성 또는 유연한 데이터 처리에 많은 도움이 될 것 같다는 점도 첨언하고 싶습니다!!
